### PR TITLE
add new collections for consideration

### PR DESCRIPTION
**Pull request by wilsoncusack**
paprMEME launched with 10 NFT collections allowlisted, but the stated criteria
- popular but underserved
- durable value
- fun to experiment with
- likely to be held by those who might want to try papr

could allow for many more. Additionally, so far paprMEME rates have been relatively low or negative and so expanding the set of borrowers could make sense. 

The above criteria, taken from the README and roughly equivalent to what appears on www.papr.wtf, are fairly imprecise. It would be easier if the token was targeted at a certain category of NFT--trading volume, age, bid depth, etc.--and maybe over the long term paprMEME will define itself more precisely according to those terms. The collections that are proposed for consideration in this PR to varying degrees meet the following criteria:

- highly traded
- strong community
- commonly used for loans but not well serviced by a peer to protocol/instant loan model 
- Are at least a few months old 
- have decent collection bid depth (collection bid depth can be thought of as an approximation for the likelihood of liquidation shortfall. It can help answer the question, "How many of these NFTs could be auctioned at the same time before liquidations start to shortfall?" Where shortfall is created by an auction yielding less than 50% of the current top collection bid.)

The community can choose which collections make sense based on the recommendations and have their say in the selection process by passing along comments.

### CODE REVIEW

**Comment by wilsoncusack** [comment]
Easier to read the file here https://github.com/with-backed/paprMEME-info/blob/wilson/propose-in-consideration/in-consideration.md

File: in-consideration.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+|World of Women|0xe785e82358879f061bc3dcac6f0444462d4b5330|2021-07-27|10000|1 Day: 33.1<br/>7 Day: 523.52<br/>30 Day: 1610.46<br/>All Time: 82420.14|7 day time weighted average top collection bid:1.79243<br/>Number of bids above 50% of 1.79243: 2019
+|Art Gobblers|0x60bb1e2aa1c9acafb4d34f71585d7e959f387769|2022-10-31T20:20:17.821Z|2986|1 Day: 1.2<br/>7 Day: 99.65<br/>30 Day: 1066.51<br/>All Time: 50107.24|7 day time weighted average top collection bid:0.87251<br/>Number of bids above 50% of 0.87251: 1390
+|Froggy Friends Official|0x29652c2e9d3656434bc8133c69258c8d05290f41|2022-03-11T07:12:49.521Z|4444|1 Day: 1.42<br/>7 Day: 24.22<br/>30 Day: 73.58<br/>All Time: 1263.22|7 day time weighted average top collection bid:0.10825<br/>Number of bids above 50% of 0.10825: 177
+|Doodles|0x8a90cab2b38dba80c64b7734e58ee1db38b8992e|2022-02-09T21:10:23.583Z|10000|1 Day: 2707.32<br/>7 Day: 9148.59<br/>30 Day: 36105.37<br/>All Time: 249731.11|7 day time weighted average top collection bid:5.21551<br/>Number of bids above 50% of 5.21551: 22809
```
Comment (wilsoncusack) [inline in-consideration.md:14]: Doodles should probably not be in paprMEME, based on the fact that they are 
- already served by several other protocols providing instant loans 
- have a bid depth significantly above the other collections, suggesting lower risk
but they are included for consideration and discussion.

File: in-consideration.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+|Regulars|0x6d0de90cdc47047982238fcf69944555d27ecb25|2021-10-30|10000|1 Day: 0.33<br/>7 Day: 4.73<br/>30 Day: 24.99<br/>All Time: 3286.61|7 day time weighted average top collection bid:0.05148<br/>Number of bids above 50% of 0.05148: 40
```
Comment (wilsoncusack) [inline in-consideration.md:15]: Regulars bid depth maybe too low

File: in-consideration.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+| World of Women          | 0xe785e82358879f061bc3dcac6f0444462d4b5330 | 2021-07-27   | 10000        | 33.1               | 523.52             | 1610.46             | 82420.14              | 1.79215                  | 1989                                  |
+| Art Gobblers            | 0x60bb1e2aa1c9acafb4d34f71585d7e959f387769 | 2022-10-31   | 2986         | 1.2                | 99.65              | 1066.51             | 50107.24              | 0.87069                  | 922                                   |
+| Froggy Friends Official | 0x29652c2e9d3656434bc8133c69258c8d05290f41 | 2022-03-11   | 4444         | 1.42               | 24.22              | 73.58               | 1263.22               | 0.10775                  | 230                                   |
+| Doodles                 | 0x8a90cab2b38dba80c64b7734e58ee1db38b8992e | 2021-10-16   | 10000        | 2707.32            | 9148.59            | 36105.37            | 249731.11             | 5.21551                  | 22809                                 |
```
Comment (wilsoncusack) [inline in-consideration.md:14]: Doodles should probably not be in paprMEME, based on the fact that they 

- already are served by several other protocols providing instant loans
- have a bid depth significantly above the other collections, suggesting lower risk

but they are included for consideration and discussion.

File: in-consideration.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+| Regulars                | 0x6d0de90cdc47047982238fcf69944555d27ecb25 | 2021-10-30   | 10000        | 0.33               | 4.73               | 24.99               | 3286.61               | 0.05144                  | 71                                    |
```
Comment (wilsoncusack) [inline in-consideration.md:15]: Bid depth maybe too low here

**Comment by redcandlerandy** [comment]
friendship bracelets up for consideration? maybe better to wait for a future with blur bids?

**Comment by wilsoncusack** [comment]
> friendship bracelets up for consideration? maybe better to wait for a future with blur bids?

I think this collection is reasonable, stats wise, but we have no good way to support art blocks since they all share the same contract

**Pull request merged**